Cabal/Distribution/Simple/GHC/ImplInfo.hs

```diff
@@ -68,7 +68,7 @@ ghcVersionImplInfo ver = GhcImplInfo
   , flagPackageConf      = v <  [7,5]
   , flagDebugInfo        = v >= [7,10]
   , supportsPkgEnvFiles  = v >= [8,0,1,20160901] -- broken in 8.0.1, fixed in 8.0.2
-  , flagWarnMissingHomeModules = v >= [8,2,1]
+  , flagWarnMissingHomeModules = v >= [8,2,0,20170201]
```

Comment:
You could have written `[8,2]` as well, as @bgamari branched off GHC 8.2 in march, and the patch was merged to GHC master on Jan 20 2017; so the earliest GHC 8.2 snapshots had the version number `8.2.0.201703xx`.